src/aten/Copy.cpp

```diff
@@ -225,10 +227,13 @@ void _copy_xpu(TensorIterator& iter, bool non_blocking) {
 
   // Copy between CPU and GPU
   c10::xpu::OptionalXPUGuard device_guard;
+  enum { _H2D_, _D2H_ } copy_kind;
```

Comment:
CUDA backend is using a CUDA runtime definition for copy direction (`cudaMemcpyKind`) and pass it to CUDA runtime copy API (`cudaMemcpyAsync`).

For us, q.memcpy is transparent for copy direction. We use the enum to distinguish code-paths to handle different Application logic. Since it is used in a local scope, following common coding style in PyTorch, leave `_` for the variables to tell developers they are not used as interface.